src/deposit.cairo

```diff
@@ -325,6 +334,22 @@ mod Deposit {
                     }
                 );
         }
+
+        fn claim_rewards(ref self: ContractState, claim_data: Claim, proofs: Span<felt252>,) {
+            assert(self.is_position_open.read(), 'Position is not open');
+            assert(proofs.len() != 0, 'Proofs Span cannot be empty');
+
+            let airdrop_addr: ContractAddress =
```

Comment:
It was not by the start, but every week zkLend deploys a new contract and rewards might go there. Please, make this a parameter of `claim_rewards` method so it won't be constant.